pmlc/dialect/stripe/from_mlir.cc

```diff
@@ -616,6 +616,23 @@ void StripeBuilder::visit(util::GenericBuilder builder) {
   for (auto operand : op->getOperands()) {
     intr->inputs.push_back(get_scalar(operand));
   }
+
+  static std::unordered_set<std::string> boolean_op_names{"cmp_eq", "cmp_ne", "cmp_gt", "cmp_lt", "cmp_ge", "cmp_le"};
+
+  if (boolean_op_names.find(intr->name) != boolean_op_names.end()) {
```

Comment:
I would probably have these two lines replaced by:
if (intr->name.rfind("cmp_", 0) == 0) {
Just in case new cmp_ operations get added we won't need to revisit this code as well. It probably runs faster too - just comparison of up to 3 characters, instead of calculating hashes and doing a HT lookup.